src/main/resources/liquibase/workbench_changelog/12_5_0.xml

```diff
@@ -0,0 +1,713 @@
+			INSERT INTO role_type_permission (permission_id, role_type_id, selectable) VALUES (20,3,1);
+			INSERT INTO role_type_permission (permission_id, role_type_id, selectable) VALUES (21,3,1);
+			INSERT INTO role_type_permission (permission_id, role_type_id, selectable) VALUES (22,3,1);
+		</sql>
+	</changeSet>
+
+	<changeSet author="vanina" id="v12.5.0-21">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				select count(1)
+				from role_permission
+			</sqlCheck>
+		</preConditions>
+		<comment>
+			INSERT INTO table role_permission
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			INSERT INTO role_permission (permission_id, role_id) VALUES (1,1);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (1,5);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (10,2);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (4,2);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (5,2);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (6,2);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (10,3);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (4,3);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (5,3);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (6,3);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (14,4);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (17,4);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (18,4);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (19,4);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (20,4);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (21,4);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (22,4);
+		</sql>
+	</changeSet>
+
+
+	<changeSet author="vanina" id="v12.5.0-22">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				select count(1) from role where id = 5;
+			</sqlCheck>
+		</preConditions>
+		<comment>
+			Update table role where row is SUPERADMIN
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			UPDATE role SET editable = 0, assignable = 0 WHERE id = 5;
+		</sql>
+	</changeSet>
+
+	<changeSet author="clarysabel" id="v12.5.0-23">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<sqlCheck expectedResult="0">
+					select count(1) from users_roles
+					where role_id in (2, 3) and crop_name is null;
+				</sqlCheck>
+			</not>
+		</preConditions>
+		<comment>Expand breeder and technician roles rows to apply to all user crops, removing rows with empty crop_name</comment>
+		<sql dbms="mysql" splitStatements="true">
+			insert into users_roles (userid, role_id, crop_name)
+			select ur.userid, ur.role_id, uc.crop_name
+			from users_roles ur
+					 inner join users_crops uc on (uc.user_id = ur.userid)
+			where ur.role_id in (2, 3);
```

Comment:
Won't it be simpler to do an UPDATE statement instead of INSERT then DELETE?
```
 UPDATE user_roles ur
   INNER join users_crops uc on uc.user_id = ur.userid
        SET crop_name = uc.crop_name
  WHERE ur.role_id in (2, 3);
```